js_modules/dagster-ui/packages/ui-core/src/app/AppProvider.tsx

```diff
@@ -121,6 +122,11 @@ export const AppProvider: React.FC<AppProviderProps> = (props) => {
     statusPolling,
   } = config;
 
+  React.useEffect(() => {
+    migrateLocalStorageKeys({from: /:dagit/gi, to: ':dagster'});
+    migrateLocalStorageKeys({from: /^dagit(\.v2)?/gi, to: 'dagster'});
+  }, []);
```

Comment:
These regexes look like they should cover all of the `dagit` keys that I found in my own app state.

Reply:
Having looked over both `js-modules/dagster-ui` and `app-cloud` in internal, I think these work for everything except the two "flags" keys, currently:

- `DAGIT_FLAGS` (in app-oss)
- `CLOUD_DAGIT_FLAGS` (in app-cloud)

If we did this (add a pattern and drop the `/i` flag), I think that would hit everything (`CLOUD_DAGIT_FLAGS` is not currently matched by this PR) and preserve casing.

```
migrateLocalStorageKeys({from: /DAGIT_FLAGS$/gi, to: 'DAGSTER_FLAGS'});
migrateLocalStorageKeys({from: /:dagit/g, to: ':dagster'});
migrateLocalStorageKeys({from: /^dagit(\.v2)?/g, to: 'dagster'});
```